Test/AnalysisEngineTest.cs

```diff
@@ -97,5 +97,13 @@ public static void TestAnalysisEngineTests()
 
             engine.Run();
         }
+
+        [Test]
+        public static void TestRunSpecificPostSearchAnalysis()
+        {
+            //code coverage unit test for an unused abstract method in post search analysis
```

Comment:
this test is doodoo

Reply:
It's superficial. The method will never be used, because the post search task doesn't truly inherit everything from a MetaMorpheus task.

Reply:
Why does it exist then, if its not truly testing anything of substance?

Reply:
The base method is used for most tasks. The "PostSearchTask" does not implement the abstract method, but still must have that method to inherit from the abstract class.